test/integration/git-credentials-attributes/serverspec/git_credentials_attributes_spec.rb

```diff
@@ -1,30 +1,26 @@
-require 'serverspec'
-
-set :backend, :exec
-
 describe package 'git' do
   it { should be_installed }
 end
 
 describe file '/root/.git-credentials' do
   it { should be_file }
-  it { should be_owned_by 'root' }
-  it { should be_grouped_into 'root' }
-  it { should be_mode '600' }
+  its('owner') { should be 'root' }
+  its('group') { should be 'root' }
+  its('mode') { should be '0600' }
   [
     %r{https://user:pass@example.net/hello/world.git},
     %r{ssh://foo:bar@example.org/foo/bar.git},
   ].each do |line|
-    its(:content) { should match line }
+    its('content') { should match line }
   end
 end
 
 describe file '/root/.gitconfig' do
```

Comment:
Please change this to using the [ini resource](https://www.inspec.io/docs/reference/resources/ini/). Change this on all checks for the ``.gitconfig`` file.

Reply:
Since I can't check `owner`, `group`, and `be_file` within [ini resource](https://www.inspec.io/docs/reference/resources/ini/), I split up those checks into another `describe` block with `file` resource. Is this ok?

```
describe ini('/root/.gitconfig') do
  its('credential.helper') { should eq 'store --file /root/.git-credentials'}
  its('credential.useHttpPath') { should eq 'true' }
end

describe file '/root/test/.git' do
  it { should be_directory }
end
```